game/video.hh

```diff
@@ -2,28 +2,45 @@
 
 #include "animvalue.hh"
 #include "texture.hh"
-#include "ffmpeg.hh"
+#include <deque>
+#include <future>
 #include <string>
    
 /// class for playing videos  
 class Video {
   public:
 	/// opens given video file
 	Video(fs::path const& videoFile, double videoGap = 0.0);
+	~Video();
 	void prepare(double time);  ///< Load the current video frame into a texture
 	void render(double time);  ///< Render the prepared video frame
 	/// returns Dimensions of video clip
-	Dimensions& dimensions() { return m_texture.dimensions; }
-	/// returns Dimensions of video clip
 	Dimensions const& dimensions() const { return m_texture.dimensions; }
 
   private:
-	FFmpeg m_mpeg;
-	double m_videoGap;
-	Bitmap m_videoFrame;
+	const double m_videoGap;
 	Texture m_texture;
 	double m_textureTime;
-	double m_lastTime;
+	double m_readPosition = 0;
```

Comment:
`= 0.0` maybe (yes, another pedanticism that doesn't matter).

Reply:
done